docs/config/README.md

```diff
@@ -463,6 +463,31 @@ impure_msg = "impure shell"
 pure_msg = "pure shell"
 ```
 
+## Java
+
+The `java` module shows the currently installed version of Java.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `pom.xml` file
+- The current directory contains a file with the `.java` extension
+
+### Options
+
+| Variable   | Default        | Description                                              |
+| ---------- | -------------- | -------------------------------------------------------- |
+| `symbol`   | `"☕ "`         | The symbol used before displaying the version of Java.  |
+| `style`    | `"bold RGB(166, 42, 42)"` | The style for the module.                     |
```

Comment:
The syntax for colors is `#RRGGBB`, so the style would be `bold #a62a2a`

Reply:
I think built-in modules should probably all use the 16 ANSI colors to remain compatible with as many emulators and configurations as possible.

Reply:
Sure, I can change it to be red instead.

Reply:
Didn't think about that. This is probably right. The color appears to be a darker version of the red color, perhaps `dimmed red` would be a good color?